tests/test_dashboard_page.py

```diff
@@ -25,12 +25,14 @@ def test_footer_all_elements_visible_and_clickable_tc_002(self, locator):
         assert driver.is_visible(locator)
         assert driver.is_clickable(locator)
 
-    @pytest.mark.parametrize('locator', DashboardPageLocators.locators_dashboard_all)
+    @pytest.mark.parametrize('locator', DashboardPageLocators.locators_dashboard_all,
+                             ids=DashboardPageLocators.ids_dashboard_all)
     def test_dashboard_all_element_is_visible_tc_008(self, locator):
         driver = DashboardPage(self.driver)
```

Comment:
driver = DashboardPage(self.driver)
for locator in DashboardPageLocators.ids_dashboard_all:
   assert driver.is_visible(locator)

Reply:
thank you